handlers/gke.go

```diff
@@ -26,7 +33,39 @@ func findUnusedGKECluster(clusterMap *clusters.Map, leaseMap *leases.Map) (*cont
 	return nil, errUnusedGKEClusterNotFound
 }
 
-func createKubeConfigFromCluster(cluster *container.Cluster) string {
-	// TODO: implement
-	return ""
+func createKubeConfigFromCluster(cluster *container.Cluster) ([]byte, error) {
+	contextName := strings.ToLower(cluster.Name)
+	authInfoName := strings.ToLower(cluster.Name)
```

Comment:
this seems unnecessary to call `strings.ToLower` twice here. How about just `authInfoName := contextName`?